packages/features/bookings/lib/get-booking.ts

```diff
@@ -109,24 +109,20 @@ export const getBookingWithResponses = <
 export default getBooking;
 
 export const getBookingForReschedule = async (uid: string) => {
-  let eventTypeId: number | null = null;
   let rescheduleUid: string | null = null;
-  eventTypeId =
-    (
-      await prisma.booking.findFirst({
-        where: {
-          uid,
-        },
-        select: {
-          eventTypeId: true,
-        },
-      })
-    )?.eventTypeId || null;
```

Comment:
`eventTypeId` is set to null for dynamic bookings, as they don't respond to any event type, which made the booking data to be null, triggering an odd behavior in the booker state